src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -107,6 +101,26 @@ def try_run_vipin(log_path: Path) -> None:
         logger.warning(f"Performance reporting failed with: {e}")
 
 
+def run_rq_dashboard(redis_urls: list, output_directory: Path) -> None:
+    # Get generic hostname url
+    url = redis_urls[0].split("//")[1]
+    hostname = url.split(":")[0]
+
+    # Set up log file
+    rq_dashboard_log = (output_directory / "rq.log").open("a")
+    logger.info("Fetching redis urls and starting RQ-Dashboard")
+    split_urls = " -u ".join(url for url in redis_urls)
+    command = "rq-dashboard -u " + split_urls + " --debug"
+
```

Comment:
I think this implementation could be improved depending on what we want.  Currently, all redis urls are submitted to the dashboard but it doesn't have to be that way.  Users could also open up another dashboard with just one redis db.  There are a lot of options here.

Reply:
We want all redis urls in the same dashboard, we just want to change the way the dashboard handles the information